### ISSUE TITLE
UWP store submission fails because of AppCenter

### ISSUE THREAD

**Issue by mklemarczyk** [microsoft/appcenter-sdk-dotnet#1299]
We are using Microsoft.AppCenter for Analytics and Crashes in a UWP app. The root NuGet package (Microsoft.AppCenter) has a reference to sqlite-net-pcl which causes our store submission to fail.

The certification tests on our local machine work without issues. But when we upload our package to the store, the WACK test fails with a lot of errors regarding e_sqlite3.dll:
```
API sqlite3_aggregate_context in e_sqlite3.dll is not supported for this application type. CartoonInk.dll calls this API.
API sqlite3_aggregate_count in e_sqlite3.dll is not supported for this application type. CartoonInk.dll calls this API.
API sqlite3_backup_finish in e_sqlite3.dll is not supported for this application type. CartoonInk.dll calls this API.
API sqlite3_backup_init in e_sqlite3.dll is not supported for this application type. CartoonInk.dll calls this API.
```

Why would the AppCenter package need SQLite to work? I've read in another issue that I can exclude dependencies, but is this a safe thing to do? I suppose that it is there for a reason?
I'd appreciate any help on this as this is a very urgent issue for us.

[AppCenter_storeCertificationFailed.pdf](https://github.com/microsoft/appcenter-sdk-dotnet/files/4082665/AppCenter_storeCertificationFailed.pdf)

**Comment by jwallra** [comment]
Hi @mklemarczyk - this is an issue caused by the arm64 support of the version of sqlite we are currently using.  We're looking to fix this as soon as we can, but since it's a breaking change we will need to release a version 3.0.

If you don't need arm64 support, you can remove it from your bundle and try submitting it again.  If you do need arm64 support, unfortunately you'll have to wait until we make our next release.

**Comment by adamhewitt627** [comment]
@jwallra Does that 3.0 breaking change also mean that #1264 is delayed, or is it still slotted for a January release?

**Comment by jwallra** [comment]
Unfortunately, we had to move back our release to February.  Sorry for the delay.  I'll update the original issue as well.

**Comment by albertomandlate** [comment]
Ok. When it's going to be ready?

**Comment by jaeklim** [comment]
@albertomandlate Thanks for your patience. You will see the newer release quite soon.

**Comment by guperrot** [comment]
The SDK has been released in version [3.0.0](https://github.com/microsoft/appcenter-sdk-dotnet/releases/tag/3.0.0) and has a fix for this issue.
